.github/workflows/stale.yml

```
@@ -22,7 +22,7 @@ jobs:
         days-before-close: 30
 
         # Issues or PRs with these labels will never be considered stale
-        exempt-pr-labels: up for grabs
+        exempt-pr-labels: stalebot ignore
```

This should probably be appended to the list.
```suggestion
        exempt-pr-labels: stalebot ignore,up for grabs
```

---

I thought of doing that, but we don't even have an `up for grabs` label.